src/coreclr/jit/CMakeLists.txt

```diff
@@ -653,6 +688,7 @@ add_pgo(clrjit)
 
 if (CLR_CMAKE_TARGET_ARCH_ARM64 OR CLR_CMAKE_TARGET_ARCH_AMD64)
   create_standalone_jit(TARGET clrjit_universal_arm64_${ARCH_HOST_NAME} OS universal ARCH arm64)
+  create_standalone_jit(TARGET clrjit_universal_wasm_${ARCH_HOST_NAME} OS universal ARCH wasm)
```

Comment:
I went back and forth a bit on whether to use the "more proper" `wasm32` name for the target architecture. In the JIT source (and I expect in the broader /runtime source as well) `TARGET_WASM` will always mean `TARGET_WASM32 || TARGET_WASM64` - a mismatch with `_wasm` being a `TARGET_WASM32`-only Jit. On the other hand, `wasm32` would be a mismatch with `Architecture.Wasm`. Matching the `Architecture` enum seems more important.